### ISSUE TITLE
Port simple OpenCV annotation tool

### ISSUE THREAD

**Issue by Victorlouisdg** [airo-ugent/airo-mono#78]
For the ICRA23 competition I wrote [a simple tool](https://github.com/Victorlouisdg/linen/blob/main/linen/linen/opencv/annotation.py) to collect a few simple annotations from manual clicks: 
![image](https://github.com/airo-ugent/airo-mono/assets/23340946/5f8e77f5-cc21-4c96-9d9c-fd1b30101399)

The idea is that you only need to provide a camera and an annotation specification and do a single function call:
```python
annotation_spec = {
    "keypoint": Annotation.Keypoint,
    "line_segment": Annotation.LineSegment,
    "line": Annotation.Line,
    "bounding_box": Annotation.BoundingBox,
}
annotations = get_manual_annotations(camera, annotation_spec)
```
One change that should be made is instead of using a `RGBCamera` object, the `get_manual_annotations()` function should simply take an image.

We also need to decide where this script goes. Perhaps in `airo-dataset-tools` or `airo-camera-toolkit`?

**Comment by tlpss** [comment]
Maybe to `camera-toolkit`? I think we will mostly use this to bypass perception modules. But both are fine by me as long as it is documented

**Comment by Victorlouisdg** [comment]
Implemented in PR #82.
